src/explore-education-statistics-common/src/components/Link.tsx

```diff
@@ -0,0 +1,34 @@
+import React, {
+  AnchorHTMLAttributes,
+  FunctionComponent,
+  ReactNode,
+} from 'react';
+
+type Props = {
+  className?: string;
+  to: string;
+  children: ReactNode;
+} & AnchorHTMLAttributes<HTMLAnchorElement>;
+
+type RendererType = FunctionComponent<Props>;
+
+// some default renderer
+let Renderer: FunctionComponent<Props> = ({ children }: Props) => {
+  return (
+    <span>
```

Comment:
This implementation will show up in common test snapshots and won't look good. Default implementation should be one of the following:

1) An anchor element which accepts as many of the props as possible. This could be confusing in the future if we add further modules (Links will look like they should work, but won't).
2) An exception that is thrown saying something like `registerInternalLink must be called first` with a Link implementation.

Number 2 is probably the safest bet but we will have to define a test implementation of Link for common. This would pretty much look like the anchor element that number 1 requires. This will have to then be registered in `setupTests` for common.

Reply:
At the moment this is only for the common project. If it needs expanding in the future, then it needs expanding in the future.